spec/models/location_spec.rb

```diff
@@ -19,6 +19,11 @@
     it 'has nooks and reservations' do
       expect(location).to respond_to :nooks, :reservations
     end
+
+    it 'must have a name' do
+      location.save
+      expect(location.errors.full_messages).to include("Name can't be blank")
```

Comment:
This assertion will fail if the user locale is not EN. Use this message from respective yml files (e.g. en.yml). I believe we don't have a sample code to demonstrate this in test though.